components/tools/OmeroWeb/omeroweb/webclient/static/webclient/javascript/ome.tree.js

```diff
@@ -418,16 +436,23 @@ $(function() {
 
                 // If this is a node which can have paged results then either specify that
                 // we want the specific page, or use default first page
-                if (node.type === 'dataset' || node.type === 'orphaned') {
+
+                // Disable paging for node without counter
+                var nopageTypes = ['project', 'screen', 'plate', 'tagset', 'tag'];
+                if (node.data && node.data.obj.childCount === undefined) {
```

Comment:
In this case, a Dataset without childCount wouldn't get the correct page displayed (even if we have a page set)?
Is that intended? Not sure I understand the use-case for this line.